dom.go

```diff
@@ -268,8 +271,16 @@ func (h *HTML) Restore(old ComponentOrHTML) {
 // function is not used directly but rather the elem subpackage (which is type
 // safe) is used instead.
 func Tag(tag string, m ...MarkupOrComponentOrHTML) *HTML {
+	return NamespacedTag(``, tag, m...)
```

Comment:
It's simpler/more common to represent empty string with an interpreted – rather than raw – string literal (`""`).